src/engine/platform/pokey.cpp

```diff
@@ -419,6 +449,10 @@ void DivPlatformPOKEY::setFlags(const DivConfig& flags) {
   for (int i=0; i<4; i++) {
     oscBuf[i]->rate=rate/14;
   }
+
+  if (useAltASAP)
+    altASAP=std::make_unique<AltASAP::Pokey>(chipClock, chipClock);
```

Comment:
Are you sure this won't cause issues with multiple POKEY?

Also be sure to call reset here too

Reply:
1. What issues do you have in mind?
2. I've refactored the core so that it will always be initialized even without reset.

Reply:
Wait! Never mind...